src/components/browse/BrowsePoolsPerformance.tsx

```diff
@@ -43,21 +45,42 @@ export type BrowsePoolsPerformanceProps = {
 export default function BrowsePoolsPerformance(
   props: BrowsePoolsPerformanceProps
 ) {
+  const [globalStats, setGlobalStats] = useState<GlobalStats>();
+
+  useEffect(() => {
+    let mounted = true;
+    const fetchGlobalStats = async () => {
+      const res = await axios.get('http://34.94.221.78:3000/global_stats');
+      const data = res.data[0];
+      if (mounted) {
+        setGlobalStats({
+          poolCount: data['pool_count'],
+          users: data['users'],
+          totalValueLocked: data['total_value_locked'],
+        });
+      }
+    }
+    fetchGlobalStats();
+    return () => {
+      mounted = false;
+    }
+  }, []);
+
   return (
     <Wrapper>
       <div className='flex flex-col gap-3 flex-grow p-4'>
         <Text size='L' weight='medium' color={METRIC_LABEL_TEXT_COLOR}>Total Value Invested</Text>
```

Comment:
I know it's not part of this PR, but do you think "Total Value Invested" is appropriate, given that people may _invest_ amount X -- but then it grows to amount Y, and this is displaying Y. Industry standard is to call this "Assets Under Management," but that may not be user-friendly. Could also say something like "Total Value Managed". What do you think?

Reply:
@adamegyed thoughts on these options?

Reply:
I think **Total Value Managed** is the best choice here.